git-link.el

```diff
@@ -35,6 +35,9 @@
 
 ;;; Change Log:
 
+;; 2020-03-16 - v0.8.4
+;; * Add support for Opendev
```

Comment:
Let's remove this. It's now March 20th and the next release will have a couple more changes that just this.

Reply:
done.